README.md

```diff
@@ -35,8 +35,107 @@ Network configuration
 ---
 
 Firecracker, by design, only supports Linux tap devices. The SDK
-provides facilities to attach a tap device to the Firecracker VM, but
-the client is responsible for further configuration.
+provides facilities to:
+* Attach a pre-created tap device, optionally with static IP configuration, to
+  the VM. This is referred to as a "static network interface".
+* Create a tap device via [CNI](https://github.com/containernetworking/cni) plugins, 
+  which will then be attached to the VM automatically by the SDK. This is referred 
+  to as a "CNI network interface"
```

Comment:
Considering CNI stands for "Container Network Interface", maybe we can refer to this as a "CNI-configured interface" instead?